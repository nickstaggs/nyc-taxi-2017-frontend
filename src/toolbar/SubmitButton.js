import React from 'react';
import './../App.css';
import Button from '@material-ui/core/Button';

const styles = {
    fontFamily: 'inherit'
}

function SubmitButton(props) {

    return (
        <Button style={styles} onClick={props.onClick}>
            <p>Submit</p>
        </Button>
    );
}

export default SubmitButton;