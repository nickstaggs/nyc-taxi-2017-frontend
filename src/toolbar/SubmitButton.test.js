import React from 'react';
import { shallow } from 'enzyme';
import SubmitButton from './SubmitButton'

it('should render', () => {
    const view = shallow(<SubmitButton />);

    expect(view).toMatchSnapshot();
});