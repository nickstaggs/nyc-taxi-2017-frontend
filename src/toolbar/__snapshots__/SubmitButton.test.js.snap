// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<WithStyles(Button)
  style={
    Object {
      "fontFamily": "inherit",
    }
  }
>
  <p>
    Submit
  </p>
</WithStyles(Button)>
`;
